tests/std/tests/LWG3987_including_flat_foo_doesnt_provide_begin_end/test_inplace_vector.cpp

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation.

```

Comment:
I think we should drop this file at this moment. It should be added when implementing `inplace_vector`.